lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,51 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+
+.nx-tabs {
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin: 0 $nx-spacing-md -1px 0;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
+  text-align: center;
+  min-width: 76px;
 
   &.active {
-    border-bottom-color: $nx-background-color-interactive;
+    @include semi-bold();
+
+    border-bottom-color: #045dd1;
+    color: #045dd1;
+  }
+
+  &:hover {
+    @include semi-bold();
+  }
+
+  &:focus {
+    border-bottom-width: 1px;
+    box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+    outline: solid 1px #02a6d9;
+  }
+
+  a:hover, a:focus {
     font-weight: bold;
```

Comment:
Any reason the mixin isn't being used here? (personally I think we should get rid of the mixins and just use `font-weight` everywhere)

Reply:
> Any reason the mixin isn't being used here? (personally I think we should get rid of the mixins and just use `font-weight` everywhere)

No good reason: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/fe3e79d3cd8a31b9d93f46ae6dafdecee940e3fa

I think there is a historical reason why the mixing were needed, that might not be true anymore